metadata-comparison/src/main/java/org/codice/nitf/metadatacomparison/FileComparer.java

```diff
@@ -213,12 +218,12 @@ private void outputBaseMetadata() throws IOException, NitfFormatException {
         }
 
         out.write("Metadata:\n");
-        for (String key : metadata.keySet()) {
-            out.write(String.format("  %s=%s\n", key, metadata.get(key)));
+        for (Map.Entry<String,String> entry : metadata.entrySet()){
+            out.write(String.format(" %s=%s%n", entry.getKey(), entry.getValue()));
```

Comment:
The `entrySet` part is a good fix, but not the space removal. See previous comment on matching GDAL.

Reply:
Will fix :+1: